README.md

```diff
@@ -1,56 +1,138 @@
+```shell
+npm install @frctl/fractal
+```
 
-## Demo
+Then create your `fractal.js` file in the project root, and configure using the [official documentation][docs].
 
-There is no 'official' demo for Fractal up yet, but [bits.24ways.org](http://bits.24ways.org) (repository: https://github.com/24ways/frontend) is an excellent example of a component library built on Fractal. Kudos to [@paulrobertlloyd](https://github.com/paulrobertlloyd) for his great work!
+Then you can either run `node_modules/.bin/fractal start` to start up the project, or create an alias under the `scripts` section in your package.json as a shortcut.
 
-## Requirements
+e.g.
 
-Any LTS (Long-term stable) version of NodeJS. At the time of writing this is 6 (Boron) and 8 (Carbon).
+```json
+scripts: {
+    "fractal:start": "fractal start --sync",
+    "fractal:build": "fractal build"
+}
+```
 
-If you're on an earlier (unsupported) version of NodeJS, then see the below compatibility table.
+then
 
-| NodeJS version | Fractal version |
-| -------------- | --------------- |
-| 4.x            | 1.1.7           |
-| 5.x            | 1.1.7           |
+```shell
+npm run fractal:start
+```
 
-## Testing
+### Installing globally
 
-Fractal is a project that has recently evolved rapidly and organically from a proof-of-concept prototype into a more stable, mature tool. Because of this it's currently pretty far behind where it should be in terms of test coverage. Any contributions on this front would be most welcome!
+```shell
+npm i -g @frctl/fractal
+```
 
-Existing tests can be run using the `npm test` command.
+This will also give you global access to the `fractal` command which you can use to scaffold a new Fractal project with `fractal new`.
+
+The downside is that it's then difficult to use different Fractal versions on different projects.
+
+This option is not recommended until a global Fractal install is capable of offloading to a project specific version.
+
+## Examples
+
+While there are no official examples or demo instances, there are lots of public examples on the [Awesome Fractal](https://github.com/frctl/awesome-fractal) repo.
 
 ## Contributing
 
 Fractal has an active group of contributors but we are always looking for more help. If you are interested in contributing then please come and say hi on [Fractal's Discord server](https://discord.gg/vuRz4Yx) - or of course we will always welcome pull requests on any of the [frctl organisation](https://github.com/frctl) repos.
 
 Please note we have a [code of conduct](.github/CODE_OF_CONDUCT.md), please follow it in all your interactions with the project.
 
-## Credits
+### Testing
+
+Fractal is a project that has recently evolved rapidly and organically from a proof-of-concept prototype into a more stable, mature tool. Because of this it's currently pretty far behind where it should be in terms of test coverage. Any contributions on this front would be most welcome!
```

Comment:
ha, maybe we should remove "recently" here, it's been quite a few years

Reply:
Done